### PR TITLE
PHP Example

### PR DESCRIPTION
**Pull request by tagadvance**

### CODE REVIEW

File: php/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+update:
+	git pull
+	bin/composer.phar update
+
+start:
+	php -S localhost:8888 -t src/
+
+test:
+	./vendor/bin/phpunit tests
```
Comment (kevindurb) [inline php/Makefile:15]: can we add all these as `PHONY` targets please?

File: php/src/Ccb/Api.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+
+namespace Ccb;
```
Comment (kevindurb) [inline php/src/Ccb/Api.php:3]: Can we change the namespace to `CCB` please?

File: php/src/Ccb/OAuth2.php [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+	public function createRefreshToken(string $refreshToken): OAuth2Credentials
+	{
+		try {
+			$credentials = $this->postJson(
+				self::TOKEN_SERVER_URL,
+				[
+					'grant_type' => 'refresh_token',
+					'refresh_token' => $refreshToken,
+				]
```
Comment (kevindurb) [inline php/src/Ccb/OAuth2.php:70]: i believe the refresh token call also requires the client id and secret

File: php/src/index.php [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+{
+	header("Location: $authorizationUrl");
+
+	print <<<HTML
+	<html>
+	<head>
+	<meta http-equiv="refresh" content="0; URL=$authorizationUrl" />
+	</head>
+	<body>
+	Your browser should redirect you momentarily. If that doesn't happen please consider upgrading to a browser that works...
```
Comment (kevindurb) [inline php/src/index.php:62]: lol a little snarky?

**Review by kevindurb — CHANGES_REQUESTED** [review]
This looks fantastic! just a couple base level changes requested!

File: php/Makefile (hunk 1 above)
Comment (tagadvance) [reply php/Makefile:15]: Well I learned something new today. Cool!

File: php/src/index.php (hunk 4 above)
Comment (tagadvance) [reply php/src/index.php:62]: Should I change it to "please consider using Firefox?" *wink* *wink*

File: php/src/Ccb/OAuth2.php (hunk 3 above)
Comment (tagadvance) [reply php/src/Ccb/OAuth2.php:70]: The `auth` config in the `postJson` method seems to work in lieu of the client id and secret. 🤷

**Pull request merged**